models/issue.go

```diff
@@ -973,7 +1028,11 @@ func newIssue(ctx context.Context, doer *user_model.User, opts NewIssueOptions)
 	if opts.IsPull {
 		_, err = e.Exec("UPDATE `repository` SET num_pulls = num_pulls + 1 WHERE id = ?", opts.Issue.RepoID)
 	} else {
-		_, err = e.Exec("UPDATE `repository` SET num_issues = num_issues + 1 WHERE id = ?", opts.Issue.RepoID)
+		if opts.Issue.IsPrivate {
+			_, err = e.Exec("UPDATE `repository` SET num_private_issues = num_private_issues + 1 WHERE id = ?", opts.Issue.RepoID)
```

Comment:
Does `ChangeConfidential` affect the `num_private_issues` counting?

Reply:
It will now ;)